app/server.jsx

```diff
@@ -113,6 +113,11 @@ export default function render(req, res) {
               ${header.link.toString()}
             </head>
             <body>
+              <script>
+                if ('serviceWorker' in navigator) {
+                  navigator.serviceWorker.register('/sw.js')
```

Comment:
Should this script be placed at the beginning of the body tag?
I've been around hunting for examples, but the service worker script mostly gets bundled with the site scripts (take a look at [Google IO](https://events.google.com/io2016/)) I believe at the bottom of the page?